lib/ts.py

```diff
@@ -1,17 +1,52 @@
+class TS(object):
+    """A class for representing time series.
 
+    Remarkably close to just being a list.  But has mean() and median().
 
-class TS(object):
+    Set accuracy = 3 to get rounding to third decimal.
+    """
 
-    def __init__(self):
+    def __init__(self, data = None, accuracy=None):
         self.data = []
-
+        self.accuracy = accuracy
+        if data:
+            for x in data:
+                self.append(x)
 
     def __len__(self):
         return len(self.data)
 
     def append(self , d):
+        if self.accuracy:
+            d = round(d, self.accuracy)
```

Comment:
The TS class has a [test](https://github.com/FriskByBergen/RPiParticle/blob/master/tests/test_ts.py) which could be extended.